lib/OpenQA/WebAPI/Controller/API/V1/JobGroup.pm

```diff
@@ -182,10 +182,14 @@ sub _validate_common_properties ($self) {
     $validation->optional('default_keep_important_logs_in_days')->num(0);
     $validation->optional('default_keep_results_in_days')->num(0);
     $validation->optional('default_keep_important_results_in_days')->num(0);
+    $validation->optional('default_keep_jobs_in_days')->num(0);
+    $validation->optional('default_keep_important_jobs_in_days')->num(0);
     $validation->optional('keep_logs_in_days')->num(0);
     $validation->optional('keep_important_logs_in_days')->num(0);
     $validation->optional('keep_results_in_days')->num(0);
     $validation->optional('keep_important_results_in_days')->num(0);
+    $validation->optional('keep_jobs_in_days')->num(0);
+    $validation->optional('keep_important_jobs_in_days')->num(0);
```

Comment:
I guess at this point it could make sense to shorten this:
```perl
for my $field (qw(keep_logs_in_days keep_important_logs_in_days
  keep_results_in_days keep_important_results_in_days
  keep_jobs_in_days keep_important_jobs_in_days)) {
    $validation->optional($field)->num(0);
    $validation->optional("default_$field")->num(0);
}
```

Reply:
or even
```perl
for my $field (qw(logs important_logs results important_results jobs important_jobs)) {
    $validation->optional("keep_${field}_in_days")->num(0);
    $validation->optional("default_keep_${field}_in_days")->num(0);
}
```